### DISCUSSION TITLE
How to disable ssh on Raspberry Pi when xrdp connection established

### DISCUSSION THREAD

**Discussion by bhaijiyunus** [neutrinolabs/xrdp#2286] [Q&A] [answered]
Hi,

I have install xrdp on Raspberry Pi board, Ubuntu 18.04 installed on it . I want to disable ssh service on Raspberry Pi when I started RDP from other Window's machine. & Resume ssh service after RDP session closed.

Is there any flag/file/configurations when xrdp initialize session, so that I can poll to disable the ssh service on RPi?

**Comment by matt335672** [comment, marked as answer]
I can think of a couple of ways you could poll for this.

1) Look at the output of `loginctl` which should tell you when a session is active.
2) Look for more than one `xrdp-sesman` process.
3) Look for more than one `xrdp` process. This is different from 2) in that you will have two `xrdp-sesman` processes for the whole lifetime of the session, but only two `xrdp` processes when you are connected to the session. It depends on exactly what you want.
4) Check the output of `ss -o state established '( sport = :3389 )'` for a live connection to xrdp.

Are any of those any good?

**Reply by bhaijiyunus** [reply]
ss -o state established | grep xrdp 
Above command gives me some entries when we connect Windows RDP to xrdp.
Thanks @matt335672 👍
